README.md

```diff
@@ -188,9 +188,9 @@ The main configuration file for the operation of ORT. This configuration is main
 the ORT instance. In contrast to the configuration files in the following, this file rarely changes once ORT is
 operational.
 
-| Format | Scope  | Default location           |
-|--------|--------|----------------------------|
-| HOCON  | Global | `$ORT_CONFIG_DIR/ort.conf` |
+| Format | Scope  | Default location             |
```

Comment:
FYI, the file at `integrations/completions/ort-completion.fish` still mentions `ort.conf`. Maybe @MarcelBochtler can comment?

Reply:
I have added a commit that updates the scripts.